src/styles/main.scss

```diff
@@ -1,3 +1,67 @@
-body {
-  margin: 0;
+$daysOfWeek: mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6;
+
+$sizeCellDay: 100px;
+$sizeGap: 1px;
+
+.page {
+  &__body {
+    margin: 0;
+  }
+}
+
+.body {
+  &__calendar {
+    margin: 0 auto;
+  }
+}
+
+.calendar {
+  display: flex;
+  width: calc((#{$sizeCellDay} * 7) + (#{$sizeGap} * (7 - 1)));
+  padding: 10px;
+
+  flex-wrap: wrap;
+  gap: $sizeGap;
+
+  &__day {
+    display: flex;
+    box-sizing: border-box;
+    width: $sizeCellDay;
+    height: $sizeCellDay;
+    border: 1px solid black;
+
+    justify-content: center;
+    align-items: center;
+
+    background-color: #eee;
+    transition: all 0.5s;
+
+    &:hover {
+      cursor: pointer;
```

Comment:
Don't use transition "all", specify to what properties it should apply.
![image](https://github.com/mate-academy/layout_calendar/assets/63243136/11e6c787-b5d8-4a2a-b856-d3e36103ebf0)
Why cursor: pointer; only for :hover pseudoclass?